test/assert/awaitility.go

```diff
@@ -23,7 +23,7 @@ import (
 const (
 	CheInstallationTimeout = time.Minute * 20
 	OperatorTimeout        = time.Second * 120
-	Timeout                = time.Second * 180
+	Timeout                = time.Second * 180 * 10
```

Comment:
I have three things for this line:
* you could use `time.Minute * 30` instead
* I believe that installation of the Tekton should be much faster - eg. within 5 or 10 minutes for sure
* Please, use dedicated constant for that particular case - the `Timeout` is used across all wait functions

Reply:
@MatousJobanek thanks, yeah, i'm going to undo this change, it won't be needed now, had it there when i was debugging. thanks for pointing it out!